lib/DBIx/Class/ResultSource.pm

```diff
@@ -702,7 +702,7 @@ sub sequence {
 
 =over 4
 
-=item Arguments: $name?, \@colnames
+=item Arguments: $name?, \@colnames, \%extra?
 
```

Comment:
Along these lines yes. Again - look at how `add_column(s)` does things, model on that.